packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 2.0.1
+version: 2.0.2
```

Comment:
Probably not necessary? Updating the version has no harm either.

Reply:
I thought more of the example was visible on pub.dev than it is.

I think it's worth mentioning in CHANGELOG, so I'll leave it. (Something I think we should consider is a way to add CHANGELOG lines for cases where it's not worth actually publishing yet. E.g., allow for "<NEXT>" as a version as long as the version in pubspec hasn't changed.)

Reply:
Dart people do `-dev` releases for these cases, until they need to publish for real, and then they collapse everything into a next version with multiple entries.